app/components/nav-bar.js

```diff
@@ -9,6 +9,24 @@ export default class NavBar extends Component {
     return !(String(this.session.currentRouteName).includes('public'));
   }
 
+  @computed('session.currentRouteName')
+  get isNotExplorePageRoute() {
+    return !(String(this.session.currentRouteName).includes('explore'));
+  }
+
+  @action
+  handleKeyPress() {
+    if (event.code === 'Enter') {
+      this.sendAction('search');
+      document.querySelector('#mobile-bar').classList.remove('show-bar');
+    }
+  }
+
+  @action
+  toggleSearchBar() {
+    document.querySelector('#mobile-bar').classList.toggle('show-bar');
```

Comment:
Why not using ember for this? Merging this as it has been too long, but please create a follow up PR to change DOM manipulation to ember